addons/web/static/src/views/list/list_renderer.js

```diff
@@ -88,6 +91,10 @@ export class ListRenderer extends Component {
         this.uiService = useService("ui");
         this.notificationService = useService("notification");
         this.keyOptionalFields = this.createKeyOptionalFields();
+        this.keyDebugOpenView = this.keyOptionalFields.replace(
+            "optional_fields",
+            "debug_open_view"
+        );
```

Comment:
```suggestion
        const key  = this.createViewKey();
        this.keyOptionalFields = `optional_fields,${key}`;
        this.keyDebugOpenView = `debug_open_view,${key}`;
```

Reply:
do you need both? Did you consider adding the open_view info in the value of the optional_fields key?

Reply:
DAM is not a big fan of adding the info into the optional fields since it's completely unrelated

Reply:
> completely unrelated
That's debatable, it's in the optional fields dropdown, so...